src/pages/stage-page/StagePage.tsx

```diff
@@ -70,8 +71,7 @@ export const StagePage = ({
           error={
```

Comment:
Sorry, not your PR but I've just noticed but the line above this (the `name` prop), should be using `${endEvent.developmentPlanEvent}-end-date`

![image](https://github.com/digital-land/local-plans-timetable/assets/29425751/6ccf51cf-bc77-4f60-985d-c9fce35b9c0b)